service/src/main/java/bio/terra/cbas/BatchAnalysisWebApplication.java

```diff
@@ -1,28 +0,0 @@
-package bio.terra.cbas;
-
-import bio.terra.common.logging.LoggingInitializer;
-import org.springframework.boot.SpringBootConfiguration;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
-import org.springframework.boot.builder.SpringApplicationBuilder;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.ComponentScan.Filter;
-import org.springframework.context.annotation.FilterType;
-
-// @SpringBootConfiguration
-@EnableAutoConfiguration
-@ComponentScan(
-    basePackages = {
-      "bio.terra.cbas",
-      "bio.terra.common.logging",
```

Comment:
Did you really intend to delete this entire file? Was it just completely unnecessary?

Reply:
I did mean to delete this. From my observation it looked like it was the original file to run the cbas app. Somewhere along the way it looks like the `App.java` file replaced it? The `App.java` file seemed to be doing everything this file was doing and more, so I opted to just get rid of it all together. I'm able to run cbas and run tests with this app deleted but I'm definitely not opposed to someone double checking in their local setup.